lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_hosts/recipes/default.rb

```diff
@@ -16,23 +16,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+START = 60
 node['coopr']['cluster']['nodes'].each do |n, v|
   short_host = v.hostname.split('.').first
-  next unless v.key?('ipaddresses') && v['ipaddresses'].key?('access_v4')
-  hostsfile_entry v['ipaddresses']['access_v4'] do
-    hostname v.hostname
-    aliases [ short_host ]
-    unique true
-    priority 62
-    action :create
-  end
-  next unless v.key?('ipaddresses') && v['ipaddresses'].key?('bind_v4')
-  next if v['ipaddresses']['bind_v4'] == v['ipaddresses']['access_v4']
```

Comment:
just noting that we are now depending on hostsfile to prune duplicates, which is fine, but something we should keep in mind